src/System.Private.ServiceModel/tools/scripts/CleanUpHttpsPort.cmd

```diff
@@ -0,0 +1,32 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+echo Remove Https configuration
+
+net session > NUL
+if NOT [%_ErrorLevel%]==[0] (
+    set __EXITCODE=%_ErrorLevel%
+    echo Error: this script must be called in an elevated window
+	goto :done
+)
+
+call :RemoveHttps 0.0.0.0:8084
+
+call :RemoveHttps 0.0.0.0:44285
+
+call :RemoveHttps 0.0.0.0:443
+
+goto :done
+
+:RemoveHttps
+netsh http delete sslcert ipport %1
```

Comment:
I think you need an "=" between ippprt and %1.  When I run it, I see:
```
WARNING: An error occurred while deleting sslcert from port 0.0.0.0:8084
The parameter is incorrect.
```
When the script is done, 'netsh http show sslcert' shows me these ports are still assigned.

When I add the "=" it (and fix the %_ErrorLevel%) this script runs successfully.